service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -48,7 +52,12 @@ public class Other implements Representable, ConfidentialParty<Other> {
     private final String detailsHiddenReason;
     private List<Element<UUID>> representedBy;
     private final String addressNotKnowReason;
-    private final IsAddressKnowType addressKnow;
+
+    // Flag for preventing from purging the converted old field value during deserialization if addressKnowV2 is null
+    @Builder.Default
+    @Getter(AccessLevel.NONE)
+    private final boolean isConvertedAddressKnow = false;
```

Comment:
Does this get used? Can see it's also defined on 173

Reply:
no, it won't get serialised to Json, as @Getter(AccessLevel.NONE) mean no getter is generated for this field

Reply:
oh! right, that one in builder is redundant

Reply:
My bad, not that one in the builder, this one is redundant. Just did some simple test, that one in builder is thread safe. so should be fine